### PR TITLE
No vuln data in popup

### PR DESCRIPTION
**Pull request by aaron-kaplan**
https://trello.com/c/ytq0V45x/87-only-admin-name-on-picker-selection

While I was in there, I also fixed a couple things that have always bugged me:
- Now you can move the marker by clicking on a spot, in addition to the existing mechanism of dragging the marker.
- Added a loading indicator to the popup so it no longer displays the name of the previously-selected region while it's waiting for the callback to finish.

### CODE REVIEW

**Review by kgraaf — APPROVED** [review]

**Comment by aaron-kaplan** [comment]
Waiting for Dan's approval on trello.

**Pull request merged**
